routing/probability_apriori.go

```diff
@@ -12,40 +12,14 @@ import (
 )
 
 const (
-	// capacityCutoffFraction and capacitySmearingFraction define how
-	// capacity-related probability reweighting works.
-	// capacityCutoffFraction defines the fraction of the channel capacity
-	// at which the effect roughly sets in and capacitySmearingFraction
-	// defines over which range the factor changes from 1 to 0.
-	//
-	// We may fall below the minimum required probability
-	// (DefaultMinRouteProbability) when the amount comes close to the
-	// available capacity of a single channel of the route in case of no
-	// prior knowledge about the channels. We want such routes still to be
-	// available and therefore a probability reduction should not completely
-	// drop the total probability below DefaultMinRouteProbability.
-	// For this to hold for a three-hop route we require:
-	// (DefaultAprioriHopProbability)^3 * minCapacityFactor >
-	//      DefaultMinRouteProbability
-	//
-	// For DefaultAprioriHopProbability = 0.6 and
-	// DefaultMinRouteProbability = 0.01 this results in
-	// minCapacityFactor ~ 0.05. The following combination of parameters
-	// fulfill the requirement with capacityFactor(cap, cap) ~ 0.076 (see
-	// tests).
-
-	// The capacityCutoffFraction is a trade-off between usage of the
-	// provided capacity and expected probability reduction when we send the
-	// full amount. The success probability in the random balance model can
-	// be approximated with P(a) = 1 - a/c, for amount a and capacity c. If
-	// we require a probability P(a) > 0.25, this translates into a value of
-	// 0.75 for a/c.
-	capacityCutoffFraction = 0.75
-
-	// We don't want to have a sharp drop of the capacity factor to zero at
-	// capacityCutoffFraction, but a smooth smearing such that some residual
-	// probability is left when spending the whole amount, see above.
-	capacitySmearingFraction = 0.1
+	// DefaultCapacityFraction is the default value for CapacityFraction. It
+	// is chosen such that the factor is active but with a small effect.
+	DefaultCapacityFraction = 0.9999
```

Comment:
Perhaps retain more fragments of the prior comments if they're still useful in explaining the derivation/rationale here?

Reply:
Some of the description is retained in the `CapacityFraction` config field docstring, see second commit. Most of the discussion around min probability has become obsolete because of the 0.5 limit, which is why I removed it.